demo/js/base64binary.js

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
3rd party code without reference? Either use a git submodule or contribute to the project to release a npm package